config/crds/migration.openshift.io_migmigrations.yaml

```diff
@@ -99,12 +104,20 @@ spec:
                   type: string
               type: object
             quiescePods:
+              description: Specifies whether to quiesce the application pods before
+                migration or not.
               type: boolean
             rollback:
+              description: Invokes the rollback migration operation, when set to true
```

Comment:
This isn't quite accurate. While `spec.canceled` causes an on-demand switchover to the `Cancel` itinerary, `spec.rollback` is only meant to be set prior to creation of a MigMigration, since a rollback is a type of migration (stage, migrate, rollback are the 3 types of migrations that can be run)